mobile/src/test/java/org/openhab/habdroid/model/OpenHABSitemapTest.java

```diff
@@ -0,0 +1,69 @@
+package org.openhab.habdroid.model;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class OpenHABSitemapTest {
+    OpenHABSitemap sitemap1;
+    OpenHABSitemap sitemap2;
+
+    @Before
+    public void initSitemaps() throws JSONException {
+        String jsonString = "{\"name\":\"demo\",\"label\":\"Main Menu\",\"link\":\"http://demo.openhab.org:8080/rest/sitemaps/demo\",\"homepage\":{\"link\":\"http://demo.openhab.org:8080/rest/sitemaps/demo/demo\",\"leaf\":false,\"timeout\":false,\"widgets\":[]}}";
+        JSONObject jsonObject = new JSONObject(jsonString);
+        sitemap1 = new OpenHAB2Sitemap(jsonObject);
+
+        jsonString = "{\"name\":\"home\",\"icon\":\"home\",\"link\":\"http://demo.openhab.org:8080/rest/sitemaps/home\",\"homepage\":{\"link\":\"http://demo.openhab.org:8080/rest/sitemaps/home/home\",\"leaf\":true,\"timeout\":false,\"widgets\":[]}}";
+        jsonObject = new JSONObject(jsonString);
+        sitemap2 = new OpenHAB2Sitemap(jsonObject);
+    }
+
+    @Test
+    public void sitemapInstanceOf() {
+        assertTrue(sitemap1 instanceof OpenHAB2Sitemap);
+        assertTrue(sitemap2 instanceof OpenHAB2Sitemap);
+    }
+
+    @Test
+    public void sitemapLabelNonNull() {
+        assertEquals("Main Menu", sitemap1.getLabel());
+        assertEquals("home", sitemap2.getLabel());
```

Comment:
Message for the assert:
"Sitemap without explicit label should return name for getLabel"